### ISSUE TITLE
Bug: bundling fails using `vite build` 

### ISSUE THREAD

**Issue by LeonHeidelbach** [bruceharrison1984/bcrypt-edge#28]
First of all, thank you so much for this project! This saved me from having to switch to a different password encryption library when deploying on cloudflare. I am however encountering an issue when trying to bundle `bcrypt-edge` directly into my project using `vite build`. When doing so I get this error message:

```console
Error [ERR_MODULE_NOT_FOUND]: Cannot find package '<path_to_project>/node_modules/bcrypt-edge/' imported from <path_to_project>/.svelte-kit/output/server/chunks/server.js
    at new NodeError (node:internal/errors:399:5)
    at legacyMainResolve (node:internal/modules/esm/resolve:235:9)
    at packageResolve (node:internal/modules/esm/resolve:876:14)
    at moduleResolve (node:internal/modules/esm/resolve:938:20)
    at defaultResolve (node:internal/modules/esm/resolve:1153:11)
    at nextResolve (node:internal/modules/esm/loader:163:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:838:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:424:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:77:40)
    at link (node:internal/modules/esm/module_job:76:36)
Emitted 'error' event on Worker instance at:
    at [kOnErrorMessage] (node:internal/worker:300:10)
    at [kOnMessage] (node:internal/worker:311:37)
    at MessagePort.<anonymous> (node:internal/worker:212:57)
    at [nodejs.internal.kHybridDispatch] (node:internal/event_target:737:20)
    at exports.emitMessage (node:internal/per_context/messageport:23:28) {
  code: 'ERR_MODULE_NOT_FOUND'
}

Node.js v18.16.0
```

Simply adding the following to my `vite.config.ts`, which removes `bcrypt-edge` from the bundle, fixes the issue completely:

```ts
export default defineConfig({
    ...
    ssr: {
        noExternal: ['bcrypt-edge']
    }
});
```

In your README you mentioned that bundling this library should not be a problem, so I assume that this is not intended.

Tested versions:
bcrypt-edge: 0.0.6
vite: vite/5.3.5 linux-x64 node-v18.16.0

**Comment by Barbapapazes** [comment]
Hey, I think, there is an issue on how this package is exported by the `package.json`. I will investigate.

**Comment by atinux** [comment]
The solution right now is to `import { hashSync } from 'bcrypt-edge/dist/bcrypt-edge'`

See https://github.com/atinux/nuxt-bcrypt-edge

**Comment by bruceharrison1984** [comment]
hmm, I wonder if this is the result of naming the files in dist. I'll see if I can work through this, it definitely isn't intended.

**Comment by Barbapapazes** [comment]
I create a SvelteKit project and successfully reproduce bug from @LeonHeidelbach. I use my fix in #29  and after that, did not see any issue during both dev and build.

https://github.com/Barbapapazes/bcrypt-edge-vitejs
